src/meshapi/urls.py

```diff
@@ -24,10 +23,8 @@
     path("query/buildings/", views.QueryBuilding.as_view(), name="meshapi-v1-query-building"),
     path("query/members/", views.QueryMember.as_view(), name="meshapi-v1-query-member"),
     path("query/installs/", views.QueryInstall.as_view(), name="meshapi-v1-query-install"),
-    path("mapdata/installs/", views.MapDataInstallList.as_view(), name="meshapi-v1-map-data-installs"),
+    path("mapdata/nodes/", views.MapDataNodeList.as_view(), name="meshapi-v1-map-data-installs"),
     path("mapdata/links/", views.MapDataLinkList.as_view(), name="meshapi-v1-map-data-links"),
     path("mapdata/sectors/", views.MapDataSectorList.as_view(), name="meshapi-v1-map-data-sectors"),
-    path("geography/whole-mesh.kml", views.map_kml, name="meshapi-v1-geography-whole-mesh-kml"),
+    path("geography/whole-mesh.kml", views.WholeMeshKML.as_view(), name="meshapi-v1-geography-whole-mesh-kml"),
```

Comment:
Interesting... I'm lazy. Can you give me a screenshot of this?

Reply:
You already approved this in a separate PR lol: https://github.com/WillNilges/meshdb/pull/158

Reply:
oh pfft